### PR TITLE
Update quickstart.rst

### PR DESCRIPTION
**Pull request by Preshh0**
This PR contains updates to make some terms and instructions in the file more understandable.

**Signed commits**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: docs/source/quick-start.rst [hunk 1]
```diff
@@ -3,13 +3,16 @@ Quickstart
...
+The instructions below are for a faster installation of Augur inorder to get to use or know the software quicker.
+
+:ref:`For a more detailed installation process, go to the "Getting Started" section. <Getting Started>`
+
 
 Setting up VirtualBox
 ~~~~~~~~~~~~~~~~~~~~~~~
 - Go to the official box website to download using this `link <https://www.virtualbox.org/>`_.
 - Download VirtualBox for "Windows hosts".
-.. image:: development-guide/images/A2.png
```
Comment (sgoggins) [inline docs/source/quick-start.rst:12]: I actually think this line is a typo, @Preshh0 ... the image name is `A2.png`

**Review by sgoggins — CHANGES_REQUESTED** [review]
Fix the typo.

File: docs/source/quick-start.rst (hunk 1 above)
Comment (sgoggins) [reply docs/source/quick-start.rst:12]: @Preshh0

**Review by sgoggins — CHANGES_REQUESTED** [review]

File: docs/source/quick-start.rst (hunk 1 above)
Comment (Preshh0) [reply docs/source/quick-start.rst:12]: Line 12?

**Pull request closed without merging**